americangut/question/__init__.py

```diff
@@ -0,0 +1,21 @@
+import os
+
```

Comment:
ya, that should be fine

On Tue, Jan 26, 2016 at 11:08 PM, J W Debelius <notifications@github.com>
wrote:

> In americangut/question/__init__.py
> <https://github.com/biocore/American-Gut/pull/188#discussion_r50908796>:
>
> > @@ -0,0 +1,21 @@
> > +import os
> > +
>
> Should the *init* document just be empty then? If so, sorry.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/American-Gut/pull/188/files#r50908796>.
>